### ISSUE TITLE
Usage of allowedContentTypes in fluidpages template

### ISSUE THREAD

**Issue by blueamerican** [FluidTYPO3/flux#1343]
When using the following code in a fluidpages template in the Configuration Section

<flux:grid.row>
<flux:grid.column colPos="0" name="main" >
<flux:form.variable name="allowedContentTypes" value="textmedia" />
</flux:grid.column>
</flux:grid.row>

and below in the Main section:

<v:content.render column="0" />

everything works fine. The Textmedia CE is predefined.

But if i have another Page Template with column"0", i also have this predefinition, although i do not have the allowedContentTypes code in there.

**Comment by blueamerican** [comment]
i have one fluidpage template with these settings:

`<flux:form.variable name="allowedContentTypes" value="textmedia,header,bullets" />`

and one template with these settings:

```
<flux:form.variable name="allowedContentTypes" value="fluidcontent_content" />
<flux:form.variable name="Fluidcontent" value="{allowedContentTypes: 'MyVendor.MyExt:SliderContainer.html'}" />
```
then i get the same settings in both templates. 
There is no separation.

The colpos of the wrapping grid is in both templates the same.
Each template has a unique form id

**Comment by blueamerican** [comment]
this still does not work in fluidpages

**Comment by NamelessCoder** [comment]
Solved as part of https://github.com/FluidTYPO3/flux/pull/1563, will be included in release 9.0. Please see the PR for more details, among other things about how to use the migration script.

**Comment by blueamerican** [comment]
In flux 9.2.0 (TYPO3 8.7.26) the predefining of content elemnts from flux container still does not work.

If you have CE "test1" with a column:

<flux:grid.column name="column1" colPos="0" label="Container">
    <flux:form.variable name="allowedContentTypes" value="mywebsite_mycontentelement" />
</flux:grid.column>

and another CE test2  with a column:
<flux:grid.column name="column2" colPos="1" label="Container">
    <flux:form.variable name="allowedContentTypes" value="mywebsite_mycontentelement1" />
</flux:grid.column>

you still get BOTH CE's as a predefinition on each container.
there is no separation!
